lib/events/auditlog.go

```diff
@@ -52,11 +52,17 @@ const (
 	// in /var/lib/teleport/log/sessions
 	SessionLogsDir = "sessions"
 
-	// StreamingLogsDir is a subdirectory of sessions /var/lib/teleport/log/streaming
+	// StreamingSessionsDir is a subdirectory of sessions (/var/lib/teleport/log/upload/streaming)
 	// is used in new versions of the uploader. This directory is used in asynchronous
```

Comment:
```suggestion
	// that is used in new versions of the uploader. This directory is used in asynchronous
```

Missing word, I think?